build.sh

```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
following previous comment: what are those options useful for?

Reply:
I used this option to resolve the error addressed in this link
[https://github.com/conda/conda/issues/508](https://github.com/conda/conda/issues/508)